pipelineRunner.go

```diff
@@ -906,16 +918,19 @@ func (pr *pipelineRunnerImpl) isFinalStageComplete(stages []*manifest.EstafetteS
 			return false
 		}
 
+		// check that all service containers are done
 		allNestedServicesAreDone := true
-		for _, s := range lastBuildLogsStep.Services {
-			switch s.Status {
-			case contracts.StatusSucceeded,
-				contracts.StatusFailed,
-				contracts.StatusSkipped,
-				contracts.StatusCanceled:
-
-			default:
-				allNestedServicesAreDone = false
+		for _, st := range pr.buildLogSteps {
```

Comment:
Make sure all services for previous stages have finished as well; with multi-stage services they can still be running otherwise.